### DISCUSSION TITLE
issue while building project at Mac-os

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1633] [Q&A] [answered]
Hi All,
I would like to contribute in this project so  i was trying to build locally using below command . it is failing while building . 
./mvnw clean install -Pprod
s@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@2473b9ce], contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map[[empty]]], class annotated with @DirtiesContext [false] with mode [null].
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.023 s <<< FAILURE! - in com.provectus.kafka.ui.KafkaConsumerGroupTests
[ERROR] shouldReturnConsumerGroupsWithPagination  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.AbstractBaseTest

[ERROR] shouldNotFoundWhenNoSuchConsumerGroupId  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerGroupTests

[ERROR] shouldOkWhenConsumerGroupIsNotActive  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerGroupTests

[ERROR] shouldBeBadRequestWhenConsumerGroupIsActive  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerGroupTests

[INFO] Running com.provectus.kafka.ui.KafkaConsumerTests
17:19:37.790 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
17:19:37.790 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
17:19:37.790 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [com.provectus.kafka.ui.KafkaConsumerTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
17:19:37.791 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [com.provectus.kafka.ui.KafkaConsumerTests]: class path resource [com/provectus/kafka/ui/KafkaConsumerTests-context.xml] does not exist
17:19:37.791 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [com.provectus.kafka.ui.KafkaConsumerTests]: class path resource [com/provectus/kafka/ui/KafkaConsumerTestsContext.groovy] does not exist
17:19:37.791 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [com.provectus.kafka.ui.KafkaConsumerTests]: no resource found for suffixes {-context.xml, Context.groovy}.
17:19:37.791 [main] INFO org.springframework.test.context.support.AnnotationConfigContextLoaderUtils - Could not detect default configuration classes for test class [com.provectus.kafka.ui.KafkaConsumerTests]: KafkaConsumerTests does not declare any static, non-private, non-final, nested classes annotated with @Configuration.
17:19:37.794 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Found @SpringBootConfiguration com.provectus.kafka.ui.KafkaUiApplication for test class com.provectus.kafka.ui.KafkaConsumerTests
17:19:37.797 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [com.provectus.kafka.ui.KafkaConsumerTests]: using defaults.
17:19:37.797 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
17:19:37.798 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [javax/servlet/ServletContext]
17:19:37.798 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@6b71e98f, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@6ed7b9c5, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@105dc246, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@1b556a88, org.springframework.test.context.support.DirtiesContextTestExecutionListener@7f8cd5fc, org.springframework.test.context.transaction.TransactionalTestExecutionListener@2c465259, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@52f7fa65, org.springframework.test.context.event.EventPublishingTestExecutionListener@62c6db99, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@7bfcc108, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@65b680b4, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@87f6ab5, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@5c316230, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@41581c3f, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@10ba9780]
17:19:37.798 [main] DEBUG org.springframework.test.context.support.AbstractDirtiesContextTestExecutionListener - Before test class: context [DefaultTestContext@63636de0 testClass = KafkaConsumerTests, testInstance = [null], testMethod = [null], testException = [null], mergedContextConfiguration = [ReactiveWebMergedContextConfiguration@3494a35e testClass = KafkaConsumerTests, locations = '{}', classes = '{class com.provectus.kafka.ui.KafkaUiApplication}', contextInitializerClasses = '[class com.provectus.kafka.ui.AbstractBaseTest$Initializer]', activeProfiles = '{test}', propertySourceLocations = '{}', propertySourceProperties = '{org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true}', contextCustomizers = set[[ImportsContextCustomizer@352ce817 key = [org.springframework.boot.autoconfigure.security.oauth2.client.reactive.ReactiveOAuth2ClientAutoConfiguration, org.springframework.boot.autoconfigure.security.oauth2.resource.reactive.ReactiveOAuth2ResourceServerAutoConfiguration, org.springframework.boot.autoconfigure.security.reactive.ReactiveSecurityAutoConfiguration, org.springframework.boot.autoconfigure.security.reactive.ReactiveUserDetailsServiceAutoConfiguration, org.springframework.boot.test.autoconfigure.web.reactive.WebTestClientAutoConfiguration]], org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@7b993c65, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@4d6025c5, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@42b3b079, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@589da3f3, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@4c012563, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@ef5b24ff, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7b7fdc8, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@2473b9ce], contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map[[empty]]], class annotated with @DirtiesContext [false] with mode [null].
17:19:37.801 [main] DEBUG org.springframework.test.context.support.AbstractDirtiesContextTestExecutionListener - After test class: context [DefaultTestContext@63636de0 testClass = KafkaConsumerTests, testInstance = [null], testMethod = [null], testException = [null], mergedContextConfiguration = [ReactiveWebMergedContextConfiguration@3494a35e testClass = KafkaConsumerTests, locations = '{}', classes = '{class com.provectus.kafka.ui.KafkaUiApplication}', contextInitializerClasses = '[class com.provectus.kafka.ui.AbstractBaseTest$Initializer]', activeProfiles = '{test}', propertySourceLocations = '{}', propertySourceProperties = '{org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true}', contextCustomizers = set[[ImportsContextCustomizer@352ce817 key = [org.springframework.boot.autoconfigure.security.oauth2.client.reactive.ReactiveOAuth2ClientAutoConfiguration, org.springframework.boot.autoconfigure.security.oauth2.resource.reactive.ReactiveOAuth2ResourceServerAutoConfiguration, org.springframework.boot.autoconfigure.security.reactive.ReactiveSecurityAutoConfiguration, org.springframework.boot.autoconfigure.security.reactive.ReactiveUserDetailsServiceAutoConfiguration, org.springframework.boot.test.autoconfigure.web.reactive.WebTestClientAutoConfiguration]], org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@7b993c65, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@4d6025c5, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@42b3b079, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@589da3f3, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@4c012563, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@ef5b24ff, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7b7fdc8, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@2473b9ce], contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map[[empty]]], class annotated with @DirtiesContext [false] with mode [null].
[ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 0.022 s <<< FAILURE! - in com.provectus.kafka.ui.KafkaConsumerTests
[ERROR] shouldReturn404ForNonExistingTopic  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.AbstractBaseTest

[ERROR] shouldReturn404ForNonExistingBroker  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerTests

[ERROR] shouldReturnConfigsForBroker  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerTests

[ERROR] shouldRetrieveTopicConfig  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerTests

[ERROR] shouldDeleteRecords  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerTests

[ERROR] shouldIncreasePartitionsUpTo10  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.provectus.kafka.ui.KafkaConsumerTests

[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.ExplainStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.ExplainStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.DescribeStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.DescribeStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.DropStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.DropStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.CreateStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.CreateStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.SelectStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.SelectStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.ShowStrategyTest
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in com.provectus.kafka.ui.strategy.ksql.statement.ShowStrategyTest
[INFO] Running com.provectus.kafka.ui.strategy.ksql.statement.TerminateStrategyTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.provectus.kafka.ui.strategy.ksql.statement.TerminateStrategyTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   KafkaConnectServiceTests.shouldFilterByNameConnectors » NoClassDefFound Could ...
[ERROR]   KafkaConnectServiceTests.shouldFilterByStatusConnectors » NoClassDefFound Coul...
[ERROR]   KafkaConnectServiceTests.shouldFilterByTypeConnectors » NoClassDefFound Could ...
[ERROR]   KafkaConnectServiceTests.shouldListAllConnectors » NoClassDefFound Could not i...
[ERROR]   KafkaConnectServiceTests.shouldListConnectors » NoClassDefFound Could not init...
[ERROR]   KafkaConnectServiceTests.shouldNotFilterConnectors » NoClassDefFound Could not...
[ERROR]   KafkaConnectServiceTests.shouldRetrieveConnector » NoClassDefFound Could not i...
[ERROR]   KafkaConnectServiceTests.shouldRetrieveConnectorPlugins » NoClassDefFound Coul...
[ERROR]   KafkaConnectServiceTests.shouldReturn400WhenConnectReturns400ForInvalidConfigCreate » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldReturn400WhenConnectReturns400ForInvalidConfigUpdate » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldReturn400WhenConnectReturns500ForInvalidConfigCreate » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldReturn400WhenConnectReturns500ForInvalidConfigUpdate » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldReturn400WhenTryingToCreateConnectorWithExistingName » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldReturnNotFoundForNonExistingCluster » ExceptionInInitializer
[ERROR]   KafkaConnectServiceTests.shouldReturnNotFoundForNonExistingConnectName » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldSuccessfullyValidateConnectorPluginConfiguration » NoClassDefFound
[ERROR]   KafkaConnectServiceTests.shouldUpdateConfig » NoClassDefFound Could not initia...
[ERROR]   KafkaConnectServiceTests.shouldValidateAndReturnErrorsOfConnectorPluginConfiguration » NoClassDefFound
[ERROR]   KafkaConsumerGroupTests.shouldBeBadRequestWhenConsumerGroupIsActive » NoClassDefFound
[ERROR]   KafkaConsumerGroupTests.shouldNotFoundWhenNoSuchConsumerGroupId » NoClassDefFound
[ERROR]   KafkaConsumerGroupTests.shouldOkWhenConsumerGroupIsNotActive » NoClassDefFound
[ERROR]   KafkaConsumerGroupTests.shouldReturnConsumerGroupsWithPagination » NoClassDefFound
[ERROR]   KafkaConsumerTests.shouldDeleteRecords » NoClassDefFound Could not initialize ...
[ERROR]   KafkaConsumerTests.shouldIncreasePartitionsUpTo10 » NoClassDefFound Could not ...
[ERROR]   KafkaConsumerTests.shouldRetrieveTopicConfig » NoClassDefFound Could not initi...
[ERROR]   KafkaConsumerTests.shouldReturn404ForNonExistingBroker » NoClassDefFound Could...
[ERROR]   KafkaConsumerTests.shouldReturn404ForNonExistingTopic » NoClassDefFound Could ...
[ERROR]   KafkaConsumerTests.shouldReturnConfigsForBroker » NoClassDefFound Could not in...
[ERROR]   KafkaTopicCreateTests.shouldCreateNewTopicSuccessfully » NoClassDefFound Could...
[ERROR]   KafkaTopicCreateTests.shouldRecreateExistingTopicSuccessfully » NoClassDefFound
[ERROR]   KafkaTopicCreateTests.shouldReturn400IfTopicAlreadyExists » NoClassDefFound Co...
[ERROR]   ReadOnlyModeTests.shouldCreateTopicForNonReadonlyCluster » NoClassDefFound Cou...
[ERROR]   ReadOnlyModeTests.shouldNotCreateTopicForReadonlyCluster » NoClassDefFound Cou...
[ERROR]   ReadOnlyModeTests.shouldNotUpdateTopicForReadonlyCluster » NoClassDefFound Cou...
[ERROR]   ReadOnlyModeTests.shouldUpdateTopicForNonReadonlyCluster » NoClassDefFound Cou...
[ERROR]   SchemaRegistryServiceTests.should404WhenGetAllSchemasForUnknownCluster » NoClassDefFound
[ERROR]   SchemaRegistryServiceTests.shouldBeBadRequestIfNoSchemaType » NoClassDefFound ...
[ERROR]   SchemaRegistryServiceTests.shouldCreateNewProtobufSchema » NoClassDefFound Cou...
[ERROR]   SchemaRegistryServiceTests.shouldOkWhenCreateNewSchemaThenGetAndUpdateItsCompatibilityLevel » NoClassDefFound
[ERROR]   SchemaRegistryServiceTests.shouldReturn404WhenGetLatestSchemaByNonExistingSubject » NoClassDefFound
[ERROR]   SchemaRegistryServiceTests.shouldReturn409WhenSchemaDuplicatesThePreviousVersion » NoClassDefFound
[ERROR]   SchemaRegistryServiceTests.shouldReturnBackwardAsGlobalCompatibilityLevelByDefault » NoClassDefFound
[ERROR]   SchemaRegistryServiceTests.shouldReturnNotEmptyResponseWhenGetAllSchemas » NoClassDefFound
[ERROR]   TailingEmitterTest.allNewMessageThatFitFilterConditionShouldBeEmitted » NoClassDefFound
[ERROR]   TailingEmitterTest.allNewMessagesShouldBeEmitted » NoClassDefFound Could not i...
[ERROR]   ConfigTest.testAlterConfig » NoClassDefFound Could not initialize class com.pr...
[ERROR]   ConfigTest.testAlterReadonlyConfig » NoClassDefFound Could not initialize clas...
[ERROR]   LogDirsTest.testAllBrokers » NoClassDefFound Could not initialize class com.pr...
[ERROR]   LogDirsTest.testChangeDirToWrongDir » NoClassDefFound Could not initialize cla...
[ERROR]   LogDirsTest.testOneBrokers » NoClassDefFound Could not initialize class com.pr...
[ERROR]   LogDirsTest.testWrongBrokers » NoClassDefFound Could not initialize class com....
[ERROR]   OffsetsResetServiceTest.failsIfGroupDoesNotExists » NoClassDefFound Could not ...
[ERROR]   OffsetsResetServiceTest.failsIfGroupIsActive » NoClassDefFound Could not initi...
[ERROR]   OffsetsResetServiceTest.resetToEarliest » NoClassDefFound Could not initialize...
[ERROR]   OffsetsResetServiceTest.resetToLatest » NoClassDefFound Could not initialize c...
[ERROR]   OffsetsResetServiceTest.resetToOffsets » NoClassDefFound Could not initialize ...
[ERROR]   OffsetsResetServiceTest.resetToOffsetsCommitsEarliestOrLatestOffsetsIfOffsetsBoundsNotValid » NoClassDefFound
[ERROR]   OffsetsResetServiceTest.resetToTimestamp » NoClassDefFound Could not initializ...
[ERROR]   RecordEmitterTest » NoClassDefFound Could not initialize class com.provectus.k...
[ERROR]   SendAndReadTests.keyIsIntValueIsDoubleShouldBeSerializedAsStrings » NoClassDefFound
[ERROR]   SendAndReadTests.keyWithAvroSchemaValueWithAvroSchemaKeyIsNull » NoClassDefFound
[ERROR]   SendAndReadTests.keyWithAvroSchemaValueWithAvroSchemaValueIsNull » NoClassDefFound
[ERROR]   SendAndReadTests.keyWithAvroSchemaValueWithNoSchema » NoClassDefFound Could no...
[ERROR]   SendAndReadTests.keyWithAvroSchemaValueWithProtoSchema » NoClassDefFound Could...
[ERROR]   SendAndReadTests.keyWithJsonValueWithJsonSchemaKeyValueIsNull » NoClassDefFound
[ERROR]   SendAndReadTests.keyWithNoSchemaValueWithAvroSchema » NoClassDefFound Could no...
[ERROR]   SendAndReadTests.keyWithNoSchemaValueWithProtoSchema » NoClassDefFound Could n...
[ERROR]   SendAndReadTests.keyWithProtoSchemaValueWithJsonSchema » NoClassDefFound Could...
[ERROR]   SendAndReadTests.noKeyAndNoContentPresentTest » NoClassDefFound Could not init...
[ERROR]   SendAndReadTests.noSchemaJsonKeyJsonValue » NoClassDefFound Could not initiali...
[ERROR]   SendAndReadTests.noSchemaKeyIsNull » NoClassDefFound Could not initialize clas...
[ERROR]   SendAndReadTests.noSchemaStringKeyStringValue » NoClassDefFound Could not init...
[ERROR]   SendAndReadTests.noSchemaValueIsNull » NoClassDefFound Could not initialize cl...
[ERROR]   SendAndReadTests.nonNullableKvWithAvroSchema » NoClassDefFound Could not initi...
[ERROR]   SendAndReadTests.primitiveAvroSchemas » NoClassDefFound Could not initialize c...
[ERROR]   SendAndReadTests.topicMessageMetadataAvro » NoClassDefFound Could not initiali...
[ERROR]   SendAndReadTests.topicMessageMetadataJson » NoClassDefFound Could not initiali...
[ERROR]   SendAndReadTests.topicMessageMetadataProtobuf » NoClassDefFound Could not init...
[ERROR]   SendAndReadTests.valueWithAvroSchemaShouldThrowExceptionArgIsNotValidJsonObject » NoClassDefFound
[ERROR]   SendAndReadTests.valueWithJsonSchemaThrowsExceptionIfArgIsNotValidJsonObject » NoClassDefFound
[ERROR]   SendAndReadTests.valueWithProtoSchemaShouldThrowExceptionArgIsNotValidJsonObject » NoClassDefFound
[INFO] 
[ERROR] Tests run: 177, Failures: 0, Errors: 81, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for kafka-ui 0.0.1-SNAPSHOT:
[INFO] 
[INFO] kafka-ui ........................................... SUCCESS [  0.261 s]
[INFO] kafka-ui-contract .................................. SUCCESS [ 30.223 s]
[INFO] kafka-ui-api ....................................... FAILURE [07:04 min]
[INFO] kafka-ui-e2e-checks ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  07:35 min
[INFO] Finished at: 2022-02-17T17:19:38-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project kafka-ui-api: There are test failures.
[ERROR] 
[ERROR] Please refer to /Users/m_645600/github/kafka-ui/kafka-ui/kafka-ui-api/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :kafka-ui-api
[m_645600@AMBUS108329 17:19:38 ~/github/kafka-ui/kafka-ui]$

**Comment by Haarolean** [comment, marked as answer]
Hi, please try building it with skipping tests.
`./mvnw clean install -Pprod -DskipTests`
